introduction.md

```diff
@@ -0,0 +1,7 @@
+Sausage cupim drumstick sirloin leberkas boudin pork loin tongue filet mignon swine. Pork chop spare ribs prosciutto, picanha brisket sausage capicola rump turkey pancetta kevin flank. Pastrami shoulder drumstick, pig jerky ball tip chuck landjaeger alcatra. Picanha burgdoggen spare ribs ribeye corned beef.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.